### PR TITLE
Reporting documentation

### PR DESCRIPTION
**Pull request by ljmcgann**
Added documentation generating reports

### CODE REVIEW

File: docs/esi-leap-reporting.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+of a resource over time.
+
+To generate a csv file with data on all usage on different nodes:
+```
+openstack lease contract list --all --long -f csv > report.csv
+```
+
+The Esi-Leap api allows users to retrieve data based on certain parameters. 
+
+For example, it is possible to generate a report based on a user's openstack project_id. To get a list of all
```
Comment (tzumainn) [inline docs/esi-leap-reporting.md:13]: This essentially views a lessee's usage, correct? If so, can we say "based on a lessee's openstack project_id"?

File: docs/esi-leap-reporting.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+```
+openstack lease contract list --owner <project_id> --all --long -f csv > report.csv
+```
+
+To generate a report based on usage over a particular date range, run the command:
+
+```
+opensack lease contract list --time-range <datetime string> <datetime string> --all --long -f csv > report.csv
+```
```
Comment (tzumainn) [inline docs/esi-leap-reporting.md:31]: Can you add some text explaining that multiple options can be combined together?

File: docs/esi-leap-reporting.md (hunk 1 above)
Comment (ljmcgann) [reply docs/esi-leap-reporting.md]: fixed

File: docs/esi-leap-reporting.md (hunk 2 above)
Comment (ljmcgann) [reply docs/esi-leap-reporting.md]: done

**Pull request merged**